cluster-scope/base/core/namespaces/opf-observatorium/namespace.yaml

```diff
@@ -4,5 +4,9 @@ metadata:
   annotations:
     openshift.io/display-name: "Operate First: Observatorium"
     openshift.io/requester: operate-first
+  labels:
+    op1st-project-owner: operate-first
+    op1st-onboarding-issue: 123
+    docs: https://www.operate-first.cloud/apps/content/observatorium/README.html
```

Comment:
@bryanmontalvan let's remove the docs url for now and once we have a basic dashboard we can work on adding more information to it later